Source/Analytics/Web.Frontend/src/components/ProjectPresence.js

```diff
@@ -154,65 +154,47 @@ class AnalyticsBanner extends Component {
-
-                        <Heading color="#009f00" size={600} paddingTop={"20px"}>
-                            {`${
-                                this.state.dataCollectors.activeDataCollectors
-                            } Active Data Collectors`}
-                        </Heading>
-                        <div className="analytics--listContainer">
-                            <UnorderedList size={500}>
-                                <ListItem>{`${
-                                    this.state.dataCollectors
-                                        .inactiveDataCollectors
-                                } Inactive`}</ListItem>
-                            </UnorderedList>
-                        </div>
-                    </div>
-                    <div className="analytics--headerPanel">
-                        <i className="analytics--headerPanelIcon fa fa-exclamation-triangle fa-5x " />
-
-                        <Heading color="#9f0000" size={600} paddingTop={"20px"}>
-                            {`${this.state.alerts.totalNumberOfAlerts} Alerts`}
-                        </Heading>
-                        <div className="analytics--listContainer">
-                            <UnorderedList size={500}>
-                                {this.state.alerts.alertsPerHealthRisk.map(
-                                    (data, index) => (
-                                        <ListItem key={index}>
-                                            {data.numberOfReports}{" "}
-                                            {data.healthRisk}
-                                        </ListItem>
-                                    )
-                                )}
-                            </UnorderedList>
-                        </div>
+                <Heading size={800} marginTop={20}>Project Presence</Heading>
+
+                <div className="analytics--headerPanelContainer" style={headerPanelContainerStyle}>
+                    <HeaderPanel 
+                        headline={`${this.state.caseReports.totalNumberOfReports} Reports`}
+                        list={this.state.caseReports.reportedHealthRisks}
+                        color="#9f0000"
+                        icon="fa-heartbeat"
+                    />
+                    
+                    <HeaderPanel 
+                        headline={`${this.state.dataCollectors.activeDataCollectors} Active Data Collectors`}
+                        list={[{inactiveDataCollectors: this.state.dataCollectors.inactiveDataCollectors, name:"Inactive Data Collectors"}]}
+                        color="#009f00"
+                        icon="fa-user"
+                    />
+                    
+                    <div>
+                        {/*<Heading size={600} color={"#9f0000"}>Reports</Heading>*/}
+                        <Map />
                     </div>
+                    
+                    
+                    {/*
```

Comment:
As this is not being used I think it can rather be removed and readded later if needed.

Reply:
Agreed